tests/test-performance.js

```diff
@@ -0,0 +1,26 @@
+var performance = require("performance");
+
+var total = 0;
+var passed = 0;
+
+function assert(actual, description) {
+    total += 1;
+    var label = "\033[1m\033[31mFAIL\033[0m";
+    if (actual === true) {
+        passed += 1;
+        label = "\033[1m\033[32mPASS\033[0m";
+    }
+    console.log(label + " - " + description);
+}
+
+var before = performance.now();
+//console.log("performance.now() before:", before);
+
+setTimeout(function() {
+    var after = performance.now();
+    //console.log("performance.now() after:", after);
+    var diff = after - before;
+    // Allow for some jitter, especially on Linux
+    assert(diff > 990 && diff < 1010, "performance.now() result over known delay");
```

Comment:
@grgustaf: Do you mean that you actually see 10ms jitter on Arduino 101, or mention possibility of that? I'd say I don't have concerns about Zephyr - it's `*RT*OS` after all. On frdm_k64f, I saw 1000 diff all the time (of course, there's always a chance to get 1001 if ticks incremented literally between few CPU instructions). When I wrote the above I mostly kept Linux in mind, that's why there's whole 10ms of jitter factor - and I'm afraid, on an overloaded system, you can get more than that. Anyway, I'll update as you suggest, and even make it 989 and 1011 to follow floating-point best practices. But I'd be really interested if you saw 10ms extra on a101.